### PR TITLE
Soteriological Purpose of  Nagarjuna's Philosophy

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/articles/soteriological-purpose-of-nagarjunas-philosophy_william-l-ames.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title: "The Soteriological Purpose of Nagarjuna's Philosophy: A Study of Chapter Twenty-three of the Mūla-madhyamaka-kārikās"
+authors:
+  - "William  L.  Ames"
+external_url: "https://journals.ub.uni-heidelberg.de/index.php/jiabs/article/view/8738/2645"
+drive_links:
+  - "https://drive.google.com/file/d/10wTSD8BvGCZxubU7CNghpencgWjJfkNa/view?usp=sharing"
+course: epistemology
+tags:
+  - emptiness
+  - nibbana
```
Comment (buddhist-uni) [inline _content/articles/soteriological-purpose-of-nagarjunas-philosophy_william-l-ames.md:11]: I think maybe vipassana instead? (more about method than result?) also, perhaps the `dialogue` tag

**Review by buddhist-uni — APPROVED** [review]
Looks good.

**Comment by JJM317** [comment]
I couldn't make changes on this since it was merged, right? Did the tags
get changed?
~Joe

**Comment by buddhist-uni** [comment]
Yeah, sorry I just went ahead and made the changes myself as I was already merging in a bunch of other items 😁

**Pull request merged**